modules/mtl/arrow-mtl-data/src/test/kotlin/arrow/mtl/EitherTTest.kt

```diff
@@ -81,3 +83,15 @@ class EitherTTest : UnitSpec() {
     }
   }
 }
+
+fun <F, L> EitherT.Companion.eqK(
+  EQKF: EqK<F>,
+  EQL: Eq<L>
+) = object : EqK<EitherTPartialOf<F, L>> {
+  override fun <R> Kind<EitherTPartialOf<F, L>, R>.eqK(other: Kind<EitherTPartialOf<F, L>, R>, EQ: Eq<R>): Boolean =
+    (this.fix() to other.fix()).let {
+      EQKF.liftEq(Either.eq(EQL, EQ)).run {
+        it.first.value().eqv(it.second.value())
+      }
+    }
+}
```

Comment:
If `EitherT` has an `eq` instance then this should be moved to where that instance is, if not it's fine to keep here I guess

Reply:
moved it to the extension file although there is no Eq